app/src/org/commcare/activities/PostRequestActivity.java

```diff
@@ -110,7 +110,7 @@ private void makePostRequest() {
         if (!hasTaskLaunched && !inErrorState) {
             SimpleHttpTask postTask;
             try {
-                postTask = new SimpleHttpTask(this, url, params, true);
+                postTask = new SimpleHttpTask(this, url, params, true, null);
```

Comment:
Sorry I'm not sure what you mean here. This line and the line you linked to are calling the same constructor with the same set of arguments. What would be the reason for overloading anything?